pkg/cluster/context.go

```diff
@@ -189,6 +198,15 @@ func (c *Context) Create(cfg *config.Config, options ...CreateOption) error {
 		return err
 	}
 
+	if !cc.InstallKubernetes {
+		fmt.Printf(
+			"Cluster creation complete. To setup Kubernetes:\n\n"+
+				"  docker exec %s-%s kubeadm init  --config=/kind/kubeadm.conf --ignore-preflight-errors=all\n",
+			cc.Name(), cc.BootStrapControlPlane().Name,
```

Comment:
would it make sense to dump the list of `init` and `join` commands for all nodes?

Reply:
I have added the link to the kubeadm documentation, so the sequence of operation is always updated/versioned